nemo_text_processing/inverse_text_normalization/hi/taggers/cardinal.py

```diff
@@ -0,0 +1,168 @@
+
+        # %% Indian numeric format simple https://en.wikipedia.org/wiki/Indian_numbering_system
+        # This only covers "standard format".
+        # Conventional format like thousand crores/lakh crores is yet to be implemented
+        graph_in_thousands = pynini.union(
+            self.graph_two_digit + delete_space + delete_thousand, pynutil.insert("००", weight=0.1),
+        )
+        self.graph_thousands = graph_in_thousands
+
+        graph_in_lakhs = pynini.union(
+            self.graph_two_digit + delete_space + pynutil.delete("लाख"), pynutil.insert("००", weight=0.1),
+        )
+
+        graph_in_crores = pynini.union(
+            self.graph_two_digit + delete_space + pynutil.delete("करोड़") | pynutil.delete("crores"),
+            pynutil.insert("००", weight=0.1),
+        )
+
+        graph_in_arabs = pynini.union(
+            self.graph_two_digit + delete_space + pynutil.delete("अरब"), pynutil.insert("००", weight=0.1),
+        )
+
+        graph_in_kharabs = pynini.union(
+            self.graph_two_digit + delete_space + pynutil.delete("खरब"), pynutil.insert("००", weight=0.1),
+        )
+
+        graph_in_nils = pynini.union(
+            self.graph_two_digit + delete_space + pynutil.delete("नील"), pynutil.insert("००", weight=0.1),
+        )
+
+        graph_in_padmas = pynini.union(
+            self.graph_two_digit + delete_space + pynutil.delete("पद्म"), pynutil.insert("००", weight=0.1),
+        )
+
+        graph_in_shankhs = pynini.union(
+            self.graph_two_digit + delete_space + pynutil.delete("शंख"), pynutil.insert("००", weight=0.1),
+        )
+
+        graph_ind = (
+            graph_in_shankhs
+            + delete_space
+            + graph_in_padmas
+            + delete_space
+            + graph_in_nils
+            + delete_space
+            + graph_in_kharabs
+            + delete_space
+            + graph_in_arabs
+            + delete_space
+            + graph_in_crores
+            + delete_space
+            + graph_in_lakhs
+            + delete_space
+            + graph_in_thousands
+        )
+        graph_no_prefix = pynini.union(
+            pynini.cross("सौ", "१००")
+            | pynini.cross("हज़ार", "१०००")
+            | pynini.cross("लाख", "१०००००")
+            | pynini.cross("करोड़", "१०००००००"),
+            pynutil.insert("graph_no_prefix", weight=2),
+        )
+
+        # self.graph_year = pynini.union(graph_in_thousands + delete_space + self.graph_hundreds, graph_zero, graph_no_prefix)
+        graph = pynini.union(
+            graph_ind + delete_space + self.graph_hundreds, graph_zero, graph_no_prefix
+        )  # graph_digit_plus_hundred,
+
+        graph = graph @ pynini.union(
+            pynutil.delete(pynini.closure("०"))
+            + pynini.difference(NEMO_HI_DIGIT, "०")
+            + pynini.closure(NEMO_HI_DIGIT),
+            "०",
+        )
+
+        labels_exception = ["शून्य", "एक", "दो", "तीन"]
```

Comment:
can we add these to a data file and read from it?

Reply:
Done.